code/datums/abilities/wizard/animal.dm

```diff
@@ -76,6 +76,9 @@ var/list/animal_spell_critter_paths = list(/mob/living/critter/small_animal/cat,
 	var/datum/abilityHolder/A
 	var/mob/living/M
 
+	var/maptext_style = "color: white !important; text-shadow: 1px 1px 3px white; -dm-text-outline: 1px black;"
```

Comment:
any reason why these aren't defined on the parent? (especially since one seems to be the same for almost all of them)

Reply:
I literally didn't consider that all the spells shared a common parent, oops. I'll fix this